AssignAll/AssignAll/AssignAllMembers/ObjectInitializerAnalyzer.cs

```diff
@@ -153,5 +155,15 @@ private static ImmutableArray<string> GetIgnoredPropertyNames(ObjectCreationExpr
                     .Select(match => match.Groups[1].Value)
                     .ToImmutableArray();
         }
+
+        private BaseObjectCreationExpressionSyntax GetObjectCreation(SyntaxNode syntaxNode)
+        {
+            BaseObjectCreationExpressionSyntax result = syntaxNode as ObjectCreationExpressionSyntax;
+
+            if (result == null)
+                result = syntaxNode as ImplicitObjectCreationExpressionSyntax;
+
+            return result;
```

Comment:
Will it not be effectively the same as
```suggestion
            return syntaxNode as BaseObjectCreationExpressionSyntax;
```
(given there are only two classes derived from `BaseObjectCreationExpressionSyntax`, according to https://learn.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.csharp.syntax.baseobjectcreationexpressionsyntax?view=roslyn-dotnet-4.3.0).

Reply:
Right, there's no need to be that specific. Thanks!